models/perf/perf_report.py

```diff
@@ -0,0 +1,129 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import pandas as pd
+import argparse
+
+
+def analyze_perf_csv(csv_file, layers, show_all):
```

Comment:
What is the point of this file? It's not used - is it an internal tool? Isn't this the point of the perf tests? If there's no need for this and people can use the perf tests, then please delete this.

Reply:
I've removed this file.